cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerClusterStatusService.java

```diff
@@ -282,6 +297,26 @@ public ExtendedHostStatuses getExtendedHostStatuses(Optional<String> runtimeVers
         return new ExtendedHostStatuses(hostStates);
     }
 
+    @Override
+    public DetailedHostStatuses getDetailedHostStatuses() {
```

Comment:
the same could apply here, these logics can be even moved into another service and use existing service there, if needed

Reply:
so i would even create a new service class for new methods in ClouderaManagerClusterStatusService in order to not confuse developers and not mix these status check solutions